hw/ip/usbdev/dv/README.md

```diff
@@ -18,71 +18,59 @@
-* [Clock and reset interface for the TL and USB domains](../../../dv/sv/common_ifs/README.md)
-* [TileLink host interface](../../../dv/sv/tl_agent/README.md)
-* USBDEV IOs
-* Interrupts ([`pins_if`](../../../dv/sv/common_ifs/README.md)
-
-### Common DV utility components
-The following utilities provide generic helper tasks and functions to perform activities that are common across the project:
-* [dv_utils_pkg](../../../dv/sv/dv_utils/README.md)
-* [csr_utils_pkg](../../../dv/sv/csr_utils/README.md)
-
-### Compile-time configurations
-None for now.
-
-### Global types & methods
-All common types and methods defined at the package level can be found in `usbdev_env_pkg`.
-Some of them in use are:
-```systemverilog
-[list a few parameters, types & methods; no need to mention all]
-```
 
-### TL_agent
-USBDEV testbench instantiates (already handled in CIP base env) [tl_agent](../../../dv/sv/tl_agent/README.md) which provides the ability to drive and independently monitor random traffic via TL host interface into USBDEV device.
+USBDEV has the following interfaces, which the testbench instantiates and connects and registers with `uvm_config_db`:
+- [Clock and reset interfaces](../../../dv/sv/common_ifs/README.md) for the USB and AON clock domains.
+- A [TileLink interface](../../../dv/sv/tl_agent/README.md).
+  USBDEV is a TL-UL device, which expects to communicate with a TL-UL host.
+  In the OpenTitan SoC, this will be the Ibex core.
+- A `usb20_block_if` representing the actual USB interface.
+  This is using a class that has been developed in parallel with `usb20_if`.
+  Eventually, they will hopefully be merged again.
+- An [alert interface](../../../dv/sv/alert_esc_agent/README.md)
+- Interrupts, modelled with the basic [`pins_if`](../../../dv/sv/common_ifs/README.md) interface.
 
-###  USB20 Agent
-The [usb20_agent](../../../dv/sv/usb20_agent/README.md) is currently a skeleton implementation.
-It does not offer any functionality yet.
+### Agents
+USBDEV has dedicated agents for two interfaces.
 
-### UVM RAL Model
-The USBDEV RAL model is created with the [`ralgen`](../../../dv/tools/ralgen/README.md) FuseSoC generator script automatically when the simulation is at the build stage.
+- The dedicated [usb20_agent](../../../dv/sv/usb20_agent/README.md), which has its own documentation.
+  This handles the USB interface itself.
 
-It can be created manually by invoking [`regtool`](../../../../util/reggen/doc/setup_and_use.md):
+- The generic [tl_agent](../../../dv/sv/tl_agent/README.md) inherited from CIP base environment.
+  This handles TileLink traffic (accessing both CSRs and packet buffers)
 
 ### Reference models
-There are no reference models in use currently.
+The only reference model for USBDEV is a RAL model for CSR reads and writes.
+For this, there is a dedicated RAL model, which is created by [`ralgen`](../../../dv/tools/ralgen/README.md) as part of the build flow.
+
```

Comment:
Nit: do we need this extra line return?

Reply:
Oops, no! Thanks. I'll drop that and then merge.